src/ctia/schemas/core.clj

```diff
@@ -0,0 +1,85 @@
+(ns ctia.schemas.core
+  (:require [ctim.schemas
+             [actor :as as]
+             [bundle :as bs]
+             [campaign :as cs]
+             [coa :as coas]
+             [common :as cos]
+             [data-table :as ds]
+             [exploit-target :as es]
+             [feedback :as feedbacks]
+             [incident :as is]
+             [indicator :as ins]
+             [judgement :as js]
+             [relationships :as rels]
+             [sighting :as ss]
+             [ttp :as ttps]
+             [verdict :as vs]
+             [vocabularies :as vocs]]
+            [flanders.schema :as fs]
+            [schema.core :as s]))
+
+;; actor
+(def NewActor (fs/->schema-tree as/NewActor))
```

Comment:
If you wanted to avoid def'ing these, you could use the memoized get-schema fn.  Not a preference, just an option.